src/scenes/QtumPrediction/components/EventCard/styles.js

```diff
@@ -37,7 +37,7 @@ const styles = (theme) => ({
   eventCardName: {
     marginBottom: theme.padding.xs.px,
     display: '-webkit-box',
-    maxWidth: '700px',
+    width: '700px',
```

Comment:
same thing, use flexbox to take up all the space. avoid setting static width/heights.